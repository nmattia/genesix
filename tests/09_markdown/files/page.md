# Some other page

Go [back](./index.md)
