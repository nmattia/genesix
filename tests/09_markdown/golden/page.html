<h1 id="some-other-page">Some other page</h1>
<p>Go <a href="index.html">back</a></p>
